rotkehlchen/tests/exchanges/test_kucoin.py

```diff
@@ -259,24 +260,34 @@ def test_deserialize_v2_trade_buy(mock_kucoin):
         'type': 'market',
         'createdAt': 1612556794259,
     }
-    expected_trade = Trade(
-        timestamp=Timestamp(1612556794),
-        location=Location.KUCOIN,
-        base_asset=A_KCS,
-        quote_asset=A_USDT,
-        trade_type=TradeType.BUY,
-        amount=AssetAmount(FVal('0.2')),
-        rate=Price(FVal('1000')),
-        fee=Fee(FVal('0.14')),
-        fee_currency=A_USDT,
-        link='601da9faf1297d0007efd712',
-        notes='',
-    )
-    trade = mock_kucoin._deserialize_trade(
+    assert mock_kucoin._deserialize_trade(
         raw_result=raw_result,
         case=KucoinCase.TRADES,
-    )
-    assert trade == expected_trade
+    ) == [SwapEvent(
+        timestamp=TimestampMS(1612556794259),
+        location=Location.KUCOIN,
+        event_subtype=HistoryEventSubType.SPEND,
+        asset=A_USDT,
+        amount=FVal('200.0'),
+        location_label='kucoin',
+        unique_id='601da9faf1297d0007efd712',
+    ), SwapEvent(
+        timestamp=TimestampMS(1612556794259),
+        location=Location.KUCOIN,
+        event_subtype=HistoryEventSubType.RECEIVE,
+        asset=A_KCS,
+        amount=FVal('0.2'),
+        location_label='kucoin',
+        unique_id='601da9faf1297d0007efd712',
```

Comment:
you can assign the unique ID to a variable and avoid repeating it, also in the mock payload